src/style.css

```
@@ -1,3 +1,72 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  width: 100%;
+  display: flex;
+  justify-content: right;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+}
+
+.nav__item {
+  text-transform: uppercase;
+}
+
+.nav__item:first-child a {
```

```suggestion
.is-active {
```
The blue text color and relative positioning should be applied only for links with active class

---

Not fixed